modules/compute/gke-job-template/main.tf

```diff
@@ -123,7 +123,7 @@ locals {
       node_count               = var.node_count
       completion_mode          = var.completion_mode
       k8s_service_account_name = var.k8s_service_account_name
-      node_pool_names          = var.node_pool_name
+      node_pool_names          = var.node_pool_names
```

Comment:
Just double checking, have you tried running a job template with multiple node pools? did it work?

Reply:
For TPUs, yes it worked. I will check for GPUs.

Reply:
Worked for GPUs as well. The kueue config should be updated to use nodeLabel `cloud.google.com/gke-accelerator: ${guest_accelerator}` instead of `cloud.google.com/gke-nodepool: ${node_pool_name}`